Document that a non-PK column shouldn't be specified as the id-column [HZ-4211] (#1014)
hazelcast/hz-docs: docs/modules/mapstore/pages/configuring-a-generic-mapstore.adoc

====

|[[id-column]]`id-column`

> |Name of the column that contains the primary key. Do not specify a column without a primary key constraint as the id-column.

Reviewer: @Edizc does it really need to be primary key constraint? Wouldn't limiting to any unique key be enough?

Reviewer 2: @frant-hartm you are right. I have tested it and limiting to any unique key is enough.

> |Name of the column that contains the primary key. A column without a primary key constraint should not be specified as the id-column.

Reviewer 3: ```suggestion
|Name of the column that contains the primary key. Do not specify a column without a primary key constraint as the id-column.
```

Reviewer 3: Can we make the statement stronger, as suggested above, or is it only a recommendation (as implied by "should")?

Author: Yes it would be better like this.
